incubator/kylo/templates/sfs/nifi.yaml

```diff
@@ -0,0 +1,275 @@
+apiVersion: apps/v1beta1
```

Comment:
Most of these can now be `apps/v1` or `apps/v1beta2`. Both of these are in all supported versions of kubernetes. This can be said for all the apps objects.